curator_app/src/views/login_form.js

```diff
@@ -19,15 +19,47 @@
 
 const m = require('mithril')
 
+const api = require('../services/api')
+const forms = require('../components/forms')
+
 /**
- * The Form for authorizing an existing user.
+ * The Form for authorizing an existing agent.
  */
 const LoginForm = {
   view (vnode) {
-    return [
-      m('.header.text-center.mb-2',
-        m('h4', 'Login Form Placeholder'))
-    ]
+    const setter = forms.stateSetter(vnode.state)
+
+    return m('.login-form', [
+      m('form', {
+        onsubmit: (e) => {
+          e.preventDefault()
+          const credentials = {
+            public_key: vnode.state.publicKey,
+            password: vnode.state.password
+          }
+          api.post('authentication', credentials)
+            .then(res => {
+              api.setAuth(res.authorization)
+              m.route.set('/')
+            })
+            .catch(api.alertError)
+        }
+      },
+      m('legend', 'Login Agent'),
+      forms.textInput(setter('publicKey'), 'Public Key'),
+      forms.passwordInput(setter('password'), 'Password'),
```

Comment:
Publickey + password seems like an odd pattern. For one it requires users to copy/paste. Is there a different credential they can use like username or email? Or if they know their private key, could they login just by pasting in their private key (no password)?